query-service-client/build.gradle.kts

```diff
@@ -7,7 +7,7 @@ plugins {
 
 dependencies {
   api(project(":query-service-api"))
-  implementation("org.hypertrace.core.grpcutils:grpc-client-utils:0.7.0")
+  implementation("org.hypertrace.core.grpcutils:grpc-client-utils:0.7.2")
```

Comment:
if upgrading this, please upgrade grpc across the project to match whichever version is in here

Reply:
Done.